.nf-core.yml

```diff
@@ -1,34 +1,41 @@
 lint:
   files_unchanged:
-    - LICENSE
     - .github/CONTRIBUTING.md
     - .github/ISSUE_TEMPLATE/bug_report.yml
     - .github/ISSUE_TEMPLATE/config.yml
     - .github/ISSUE_TEMPLATE/feature_request.yml
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/workflows/branch.yml
     - .github/workflows/linting.yml
+    - .gitignore
     - assets/email_template.html
     - assets/email_template.txt
     - assets/sendmail_template.txt
     - docs/README.md
-    - .gitignore
+    - LICENSE
 
   files_exist:
+    - .github/ISSUE_TEMPLATE/config.yml
+    - .github/workflows/awsfulltest.yml
+    - .github/workflows/awstest.yml
+    - .github/workflows/push_dockerhub.yml
+    - assets/multiqc_config.yaml
     - assets/nf-core-qbic-pipelines/rnadeseq_logo_light.png
+    - bin/markdown_to_html.r
     - conf/test_full.config
-    - docs/images/nf-core-qbic-pipelines/rnadeseq_logo_light.png
     - docs/images/nf-core-qbic-pipelines/rnadeseq_logo_dark.png
-    - bin/markdown_to_html.r
-    - .github/workflows/push_dockerhub.yml
-    - assets/multiqc_config.yaml
-    - .github/workflows/awstest.yml
-    - .github/workflows/awsfulltest.yml
+    - docs/images/nf-core-qbic-pipelines/rnadeseq_logo_light.png
     - lib/WorkflowQbic-pipelines/rnadeseq.groovy
+
   nextflow_config:
+    - manifest.homePage
+    - manifest.name
     - params.input
     - show_hidden_params
-    - manifest.name
-    - manifest.homePage
+
+  multiqc_config:
+    - report_comment # otherwise throws error: multiqc_config: 'assets/multiqc_config.yml' does not contain a matching 'report_comment'.                                                                           │
+
+  actions_ci: false # otherwise throws error: actions_ci: '.github/workflows/ci.yml' does not check minimum NF version                                                                                            │
```

Comment:
This should be fixed rather than ignored. This error message is because the Nextflow versions specified in the `ci.yml` do not match the Nextflow versions specified in the `nextflow.config`.

Reply:
use the newest of both versions